src/CLI/clitree/cli-xml/ipv4.xml

```diff
@@ -104,7 +105,7 @@ limitations under the License.
             ptype="MAC_ADDR">
         </PARAM>
         <ACTION>
-            python $SONIC_CLI_ROOT/sonic-cli-arp-neigh.py get_sonic_neigh_sonic_neigh_neigh_table arp_show.j2 IPv4 mac ${mac-addr}&#xA;
```

Comment:
Have the file name sonic-cli-neigh.py

Reply:
The filename has been replaced with sonic-cli-arp-ndp.py for the reason mentioned above.

Reply:
It has been renamed to 'sonic-cli-neighbors.py'